common/core/logicaldisplay.h

```diff
@@ -111,6 +111,19 @@ class LogicalDisplay : public NativeDisplay {
   void GetDisplayCapabilities(uint32_t *numCapabilities,
```

Comment:
Does HDR only work on DrmDisplay?
if yes, no need to add this method into Logic and Mosaic display.
you can add a default method on Native display and return false.